src/Illuminate/Log/DailyChannel.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Illuminate\Log;
+
+use Monolog\Logger;
+use Monolog\Handler\RotatingFileHandler;
+use Illuminate\Contracts\Container\Container;
+
+class DailyChannel extends Channel
+{
+    /**
+     * The default options for the channel.
+     *
+     * @var array
+     */
+    protected $defaultOptions = [
+        'file' => 'laravel.log',
+        'days' => 5,
```

Comment:
Any specific reason for 5? I suggest 7, i.e. a week.

Reply:
This is the default currently used by Laravel